src/Kunstmaan/AdminListBundle/Entity/EntityVersionLock.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Kunstmaan\AdminListBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * EntityVersionLock
+ *
+ * @ORM\Table(name="kuma_entity_version_lock", indexes={
+ *     @ORM\Index(name="nt_owner_public_idx", columns={"owner", "lockable_entity_id"}),
+ * })
+ * @ORM\Entity(repositoryClass="Kunstmaan\AdminListBundle\Repository\EntityVersionLockRepository")
+ */
+class EntityVersionLock extends \Kunstmaan\AdminBundle\Entity\AbstractEntity
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="owner", type="string", length=255)
+     */
+    private $owner;
```

Comment:
I didn't want to make a hard reference to the kunstmaan user object. I can't make a OneToMany to the fosuserbundle user. This way I don't really care what sort of user it is.